internal/pkg/service/stream/storage/level/local/encoding/encoder/csv/fastcsv/writer.go

```diff
@@ -26,7 +24,7 @@ func (w *writer) WriteRow(cols *[]any) error {
 	// Write each column
 	for n, col := range *cols {
 		// Cast the value to string
-		toWrite, err := w.toCSVValue(col)
+		toWrite, err := csvfmt.Format(col)
 		if err != nil {
 			return ValueError{ColumnIndex: n, err: err}
 		}
```

Comment:
Body column, returns `[]byte`, and there is no more conversion to string in the `csvfmt.Format`